packages/react/src/GoogleSignupForm/GoogleSignupForm.tsx

```diff
@@ -0,0 +1,42 @@
+import Stack from '@mui/joy/Stack';
+import Input from '@mui/joy/Input';
+import React from 'react';
+
+import { useNileConfig } from '../context';
+import GoogleLoginButton, { buildDefaultHref } from '../GoogleLoginButton';
+
+export default function GoogleSignupForm(props: {
+  href?: string;
+  workspace?: string;
+  database?: string;
+  placeholder?: string;
+}) {
+  const { workspace, database } = props;
+  const { basePath } = useNileConfig();
+  const placeholder = props?.placeholder ?? 'Tenant Name';
+  const baseHref =
+    props?.href ?? buildDefaultHref(basePath, workspace, database);
+  const [loginHref, setLoginHref] = React.useState<string>(baseHref);
+  return (
+    <Stack gap={2}>
+      <Stack>
+        <Input
```

Comment:
this should be in the storybook, not the component

Reply:
changed in latest commit.